corehq/apps/app_manager/views/releases.py

```diff
@@ -277,6 +283,15 @@ def release_build(request, domain, app_id, saved_app_id):
             create_build_files_for_all_app_profiles.delay(domain, saved_app_id)
         _track_build_for_app_preview(domain, request.couch_user, app_id, 'User starred a build')
 
+    if toggles.APPLICATION_RELEASE_LOGS.enabled(domain):
+        ApplicationReleaseLog.objects.create(
+            domain=domain,
+            action=ApplicationReleaseLog.ACTION_RELEASED if is_released else ApplicationReleaseLog.ACTION_IN_TEST,
+            version=saved_app.version,
+            app_id=app_id,
+            user_id=request.couch_user.get_id
+        )
```

Comment:
just thinking out loud if it makes sense to keep the build id as well. This would make it easy for us to link to the build directly from these logs, like a link to the build if someone wants to look at the app build docs from here itself.
We can always fetch it from backend using this function `get_build_by_version(domain, app_id, version, return_doc=False)` though.

@orangejenny thoughts? I remember you mentioned you had concerns about storing username in the logs since that can be fetched. This is slightly different here.

Reply:
Ya, definitely no plans of using it off hand. Cool, we can leave this for now